lib/web/mage/adminhtml/wysiwyg/widget.js

```diff
@@ -513,7 +513,7 @@ define([
          * @return {null|wysiwyg.Editor|*}
          */
         getWysiwyg: function () {
-            return wysiwyg.activeEditor();
+            return tinyMCE.get(this.widgetTargetId);
```

Comment:
Hi @molovo, thanks for collaboration. 
Where you get `tineMCE` variable? I don't see any declaration of this variable.

Reply:
Hi @VladimirZaets - the `tinyMCE` variable is attached to window when TinyMCE is initialised.

Interestingly, it looks like there's actually two instances being loaded in admin, `window.tinyMCE` and `window.tinymce`. Both are the same version.

Reply:
Hi @molovo. 
yes, I know that TinyMCE currently is in global scope ( in window object ). But it's a legacy way, currently, we have some DI that implemented by RequireJS library.  So, we shouldn't rely on the property in the global scope, we should declare dependency directly by RequireJS dependency injection mechanism

Reply:
I've fixed this in my latest commit